usaspending_api/common/etl/spark.py

```diff
@@ -54,6 +57,10 @@
     DABSSubmissionWindowSchedule,
 ]
 
+_BROKER_PROXY_TABLES = [
```

Comment:
The `_` in at the beginning of the name is used to denote that something is private since Python doesn't have private types. Can remove that since you import that below.

Reply:
changed out.